MovingBlocks/Terasology
# Hind translate

### Contains

**Hindi translation updates:**
- changed "enter-username-message" string to comply with changes as stated in #2921
- added translations for new strings (storage and telemetry mostly)

### How to test

1. Run the game
2. Switch to the Hindi language
3. See everything translated :+1:
